src/objects/Tribute__c.object

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The new object will require translation files for all the supported languages. Without them in the branch, the package will not build. @jjbennett can give you more details on the process to follow to ensure the translation files for this new object are added.

Reply:
@jjbennett - Can you explain to me what I need to do for the new objects?  I assume if I added a new setting, and additional custom labels, those need I8N as well?  @force2b suggested I reach out to you!  Thanks for any assistance you can provide!!!

Reply:
@force2b - Translation files are now present.  I also brought over the translation stubs for the newly added Multiple Tribute related labels.

Reply:
Thanks!